lucien-ios/RootViewController.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  RootViewController.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 9/25/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import UIKit
+
+final class RootViewController: UIViewController {
```

Comment:
This old version of `RootViewController` should be deleted (the whole file)

Reply:
I only see one `RootViewController`, which is in the `LoginScreen` folder. Am I missing something?